inductiva/utils/format_utils.py

```diff
@@ -40,55 +39,49 @@ def seconds_formatter(secs: float) -> str:
     return str(datetime.timedelta(seconds=round(secs)))
 
 
+#clean this function
+def apply_formatters(rows, columns, formatters):
+    data = {}
+
+    for index, column_name in enumerate(columns):
+        data[column_name] = [row[index] for row in rows]
+
+    for column_name, formatter in formatters.items():
+        if column_name in data:
+            data[column_name] = [formatter(x) for x in data[column_name]]
+
+    return data
+
+
 def get_tabular_str(
     rows,
     columns,
-    default_col_space=15,
-    override_col_space=None,
     formatters=None,
 ) -> str:
     """Converts a list of lists to a string table.
```

Comment:
there is no requirement to having rows be a list of strings. It can be anything that can be convert to a `str` or `repr` representation.
Please update the docstring acoordingly and make use it documents what the args are.

Reply:
Done